daemon/daemon.go

```diff
@@ -879,33 +879,30 @@ func (daemon *Daemon) Unmount(container *container.Container) error {
 
 // V4Subnets returns the IPv4 subnets of networks that are managed by Docker.
 func (daemon *Daemon) V4Subnets() []net.IPNet {
```

Comment:
What about refactoring `V4Subnets` and `V6Subnets` into a single function that returns both slices? They are both called in the same place: https://github.com/docker/docker/blob/584e399d9632200e6c8a636339c8aa6347a57fca/daemon/cluster/listen_addr.go#L164-L166